src/integrations/YattIntegrationHelpers.js

```diff
@@ -26,8 +26,10 @@ export default {
     });
 
     const credential = credentials?.yatt[0];
+    let formattedData = this.formatData(data);
     const options = {
-      headers: this.getHeaders(credential),
+      headers:
+        formattedData.type === "bearer" ? this.getHeaders(credential) : "",
```

Comment:
put this logic in the `getHeaders` function and look specifically for `type === "bearer"` and `type === "cookie"`